app/src/main/java/com/codesoom/assignment/App.java

```diff
@@ -1,10 +1,25 @@
 package com.codesoom.assignment;
 
+import com.codesoom.assignment.config.AppConfig;
+import com.codesoom.assignment.server.TodoServer;
+import java.io.IOException;
+
 public class App {
-    public String getGreeting() {
-        return "Hello World!";
-    }
+
 
     public static void main(String[] args) {
+        App app = new App();
+        app.runTodoServer();
+    }
+
+    public void runTodoServer() {
+        try {
+            AppConfig appConfig =new AppConfig();
```

Comment:
```suggestion
            AppConfig appConfig = new AppConfig();
```

너~무 사소하지만 `=` 연산자를 쓸 때 코딩 포맷이 조금씩 다른 것 같아요!
어떻게 해도 상관은 없겠지만 일단은 일관되게 하는 것이 중요해요!
바로 위 코드처럼 양쪽 모두 스페이스를 붙이는 것이 좋을 것 같아요.